pkg/leaderelection/config.go

```diff
@@ -17,39 +17,7 @@ limitations under the License.
 package leaderelection
 
 import (
-	"fmt"
-
-	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/util/sets"
 	kle "knative.dev/pkg/leaderelection"
 )
 
-var (
-	validComponents = sets.NewString(
```

Comment:
Do we even need these files now?

Reply:
I kept the alias in case downstream repos were using the function, and the tests validate the actual config, which is still useful.